tests/dummy/urls.py

```diff
@@ -1,5 +1,5 @@
-from django.conf.urls import patterns, include, url
+from django.conf.urls import include, url
 
-urlpatterns = patterns('',
+urlpatterns = [
```

Comment:
tuple?

Reply:
I can, but I've gone the route of making all my urlpatterns a list. They get appended to, combined, and modified enough that a tuple ends up more of an annoyance (and Django docs all default to a list)

Reply:
makes sense! it's the patterns thing that I was missing before.